ParabolInc/parabol
# fix: deleting team prevents reuse of deleted team's name

# deleting team prevents reuse of deleted team's name
User cannot create a team with the same **team name** if **team name**  if **team name** has ever been used and deleted
Fixes/Partially Fixes #6484


## Demo
https://www.loom.com/share/263bf037561244d8ad55edd75a45cbee
## Testing scenarios

- [ ] Scenario A
  - Create a new team with name, 'X'
  - Delete team 'X'
  - Notice the notification in the red box that 'X' has been archived.
  - Create another new with same name, 'X'

## Final checklist
